public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="es">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta name="theme-color" content="#ffffff" />
+    <meta name="keywords"
+      content="Software Development, Full Stack Development, Front-End Development, JavaScript Development"
+    >
+    <meta name="description"
+      content="Jesmoo is a software developer specializing in Full Stack, Front-End, and JavaScript Development.
+      Explore the expense tracking application designed with a frontend mentor template."
+    >
+    <meta name="author" content="jesmoo">
+    <meta name="robots" content="noindex">
+    <title>Jesmoo - invoice app</title>
+    <base href="/">
+    <link rel="canonical" href="https://jesmoo.invoice-app.xyz" />
+  </head>
+
+  <body>
+    <div id="toast"></div>
+    <div id="root"></div>
```

Comment:
Change the shape, since root is where the app is grabbed, and the toast appears in a popup form

```suggestion
    <div id="root"></div>
    <div id="toast"></div>
```